src/subsystems/haskell/translators/stack-lock/default.nix

```diff
@@ -77,7 +77,7 @@ in {
     utils,
     ...
   }: let
-    haskellUtils = import ./utils.nix {inherit dlib lib pkgs;};
+    stackLockUtils = import ./utils.nix {inherit dlib lib pkgs;};
     all-cabal-hashes = let
```

Comment:
See point 2 in https://github.com/nix-community/dream2nix/pull/228#issue-1330391847